src/database/models/ubi/beneficiary.ts

```diff
@@ -60,11 +60,11 @@ export function initializeBeneficiary(sequelize: Sequelize): void {
             },
             address: {
                 type: DataTypes.STRING(44),
-                references: {
-                    model: 'user',
-                    key: 'address',
-                },
-                onDelete: 'RESTRICT', // delete only if active = false, separately
+                // references: {
+                //     model: 'user',
+                //     key: 'address',
+                // },
+                // onDelete: 'RESTRICT', // delete only if active = false, separately
```

Comment:
Can we still use `include: [ model: beneficiary ]` from `user` or the opposite, without this? And if so, wouldn't it be confusing? :thinking:

Reply:
yes, we can use include because I kept the associations.
and to delete a user profile and keep the beneficiary/manager (even if the address doesn't refer to any user), I needed to remove this restriction

Reply:
Ok. Then, just remove the commented code and write a note (in comment) like `// this is associated with user table` or something like that.

Reply:
done